package.json

```diff
@@ -23,15 +24,17 @@
     "react-router-redux": "^4.0.8",
     "react-scripts": "3.4.3",
     "redux": "^4.0.5",
+    "redux-actions": "^2.6.5",
     "redux-logger": "^3.0.6",
+    "redux-saga": "^1.1.3",
     "redux-thunk": "^2.3.0",
     "reselect": "^4.0.0"
   },
   "scripts": {
     "start": "node server/server.js",
-    "build": "react-scripts build",
+    "build": "REACT_APP_API_URL=https://oink-backend.herokuapp.com react-scripts build",
     "test": "react-scripts test",
-    "dev": "react-scripts start",
+    "dev": "REACT_APP_API_URL=http://localhost:5000 react-scripts start",
```

Comment:
Looks great! 💯  Only one comment which is to add dev change to ReadMe since it is different for windows users. Don't worry i'll do that :)

Reply:
Thanks thushnica!